packages/commonwealth/server/migrations/20230912211513-discord-webhook-setting.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('Chains', 'discord_webhooks_enabled', {
```

Comment:
'discord_webhooks_enabled' is misleading as you are not actually enabling _all_ webhooks for Discord just discobot activity.